app/User.php

```diff
@@ -74,6 +74,7 @@ class User extends Authenticatable implements MustVerifyEmail
         'trial_ends_at',
         'card_brand',
         'card_last_four',
+        'is_admin',
```

Comment:
I think this should probably be in "fillable" rather than hidden. What do you think?